docs/pages/web/iconbuttonfloating.js

```diff
@@ -0,0 +1,248 @@
+          title="ARIA attributes"
+          description={`
+IconButtonFloating conveys the component behavior using iconography. IconButtonFloating requires \`accessibilityLabel\`, a text description for screen readers to announce and communicate the represented [Icon](/web/icon). In the example below, the screen reader reads: "Create Pin menu". **The label should describe the intent of the action, not the Icon itself.** For example, use "Edit board" instead of "Pencil".
+
+If IconButtonFloating is used as a control button to show/hide a Popover-based component, we recommend passing the following ARIA attributes to assist screen readers:
+
+- \`accessibilityControls\`: informs the screen reader that IconButtonFloating controls the display of an anchored Popover-based component. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityHaspopup\`: informs the screen reader that there’s a Popover-based component attached to IconButtonFloating. It populates [aria-haspopup](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityExpanded\`: informs the screen reader whether an anchored Popover-based component is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+`}
+        >
+          <MainSection.Card cardSize="lg" />
+          <MainSection.Subsection
+            title="Keyboard interaction"
+            description={`
+IconButtonFloating should be contained within the \`role="contentinfo"\` container on a page. This gives screen reader users the ability to skip any main content and go directly to the action buttons. If there are multiple IconButtonFloatings, they should all be contained within the \`role="contentinfo"\` container.`}
+          />
+          <MainSection.Card cardSize="lg" />
+        </MainSection.Subsection>
+      </AccessibilitySection>
+      <MainSection name="Localization" description="Be sure to localize `accessibilityLabel`." />
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Size"
+          columns={2}
+          description="IconButtonFloating size is only available in a single size, 56px. Keeping the size consistent will promote a cohesive Pinner experience and avoid usability issues. "
+        />
+        <MainSection.Subsection
+          title="Color"
+          description={`
+          IconButtonFloating is provided in two colors schemes. It can be presented in combinations of icon and background colors.
+
+          - Dark Gray Icon ("darkGray") + White ("white") background + $elevation-floating
+          Color combination in light-mode UI.
+
+          -  White ("white") Icon + $color-background-elevation-floating
+          Color combination in dark-mode UI.
+          `}
+        />
+        <MainSection.Subsection
+          title="Elevation"
```

Comment:
This seems very implementation-y to me. 

I don't think we need a lot of "foundations" content here. 

I think i'd be great to explain how to elevate the component: position=absolute or fixed, Some might attempt Layer!
I'd explain the implementation for elevation